internal/contentmanager/contentmanager_test.go

```diff
@@ -1,207 +0,0 @@
-package contentmanager
```

Comment:
Could we create an issue to make new tests for the new `Manager`?

Reply:
Absolutely! I had left a comment on this file about tests but looks like it didn't stick after some git-fu: https://github.com/operator-framework/operator-controller/pull/1119#discussion_r1718963199

If folks are comfortable with postponing addition of tests to a follow-up to this PR I'm happy to create a separate issue, but wanted to verify that preference wasn't for me to implement them as part of this PR.

Reply:
There are now unit tests for the underlying components, but not the `Manager` itself. i felt the logic was fairly straightforward, but if we still would like to have unit tests for it I can see if I can wiggle some in.